test/libsolidity/analysis/FunctionCallGraph.cpp

```diff
@@ -173,8 +173,8 @@ void checkCallGraphExpectations(
 
 ostream& operator<<(ostream& _out, EdgeNames const& _edgeNames)
 {
-	for (auto const& edge: _edgeNames | to<vector>() | actions::sort(std::less()))
```

Comment:
You should also remove `#include <functional>` from the top of the file. I added it just for `less()`.